### PR TITLE
[WebcamAPI] Supersaiyan

### PR DESCRIPTION
**Pull request by jin-s13**
## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: mmpose/core/visualization/effects.py [hunk 1]
```diff
@@ -3,6 +3,45 @@
 import numpy as np
 
 
+def screen_matting(img, color_low=None, color_high=None, color=None):
```
Comment (ly015) [inline mmpose/core/visualization/effects.py:6]: What about moving these codes into tools/webcam/. For example in the node.py or utils.py.

File: tools/webcam/webcam_apis/nodes/frame_effect_nodes.py [hunk 2]
```diff
@@ -20,6 +23,44 @@
     psutil_proc = None
 
 
+def is_url(filename):
```
Comment (ly015) [inline tools/webcam/webcam_apis/nodes/frame_effect_nodes.py:26]: Maybe move it into webam_apis/utils?

Comment (jin-s13) [reply tools/webcam/webcam_apis/nodes/frame_effect_nodes.py:26]: OK.

File: mmpose/core/visualization/effects.py (hunk 1 above)
Comment (jin-s13) [reply mmpose/core/visualization/effects.py]: Done.

**Pull request merged**